README.mess

```diff
@@ -32,3 +32,13 @@ ldapper import <(slapcat -b "dc=example,dc=com")
 (note the ``--dry-run`` flag for that)
 
 And that's pretty much it. For more information, see the help as described above.
+
+## Docker build instructions
+
+::
+docker build -t ldapper .
+
+docker run --rm -it -v "$PWD:/app" ldapper
+
+sbcl --eval '(require "asdf")' --eval '(asdf:load-asd "/app/ldapper.asd")' --eval '(ql:quickload :ldapper)' --eval '(asdf:make :ldapper)'
+::
```

Comment:
Sorry, this is just my ignorance as someone that's never used docker before, but can't this step be part of the dockerfile?

Reply:
No, totally valid question.

I think it's a bit of a philosophical one - how you want to use it.

What you describe would be like "I run one command and a compiled binary falls out" and what I made is more like "I can use this Docker container as my complete development environment, and either run sbcl inside to build, or start a repl".

We can change that so you run one command and the build is done, and then the people can of course still interactively use the container.

Reply:
I think what I described would be more in line with what most folks would expect, and it would make the instructions look simpler.